understanding/21/non-text-contrast.html

```diff
@@ -396,6 +396,19 @@ <h4>Required for Understanding</h4>
 					<li> The information is available in another form, such as in a table that follows the graph, which becomes visible when a "Long Description" button is pressed.</li>
 					<li> The graphic is part of a logo or brand name (which is considered &quot;essential&quot; to its presentation).</li>
 				</ul>
+
+				<div class="note">
+					<p>Logos and logotypes are exempted from contrast requirements when they are purely used as
+						<em>graphical objects</em>, under the assumption that they must comply with stricter color choices mandated
+            by corporate identity guidelines.
+						However, this is not the case when they also act as <em>user interface components</em>
+						(such as links or other interactive controls). In these cases, authors should choose
```

Comment:
I think people might read your writing as best practice or advice. "If the logo is a user interface component, you should ...."
While I think you meant to say something like: "When logos are used as interface components don't meet contrast requirements, you could do one of the following..."
I think a more explicit disctinction between the clarifcation (user interface components are not excempt) and the possible solutions would be more clear than the current proposal.

Reply:
i intended it in the RFC sense

> [3](https://datatracker.ietf.org/doc/html/rfc2119#section-3). SHOULD   This word, or the adjective "RECOMMENDED", mean that there
   may exist valid reasons in particular circumstances to ignore a
   particular item, but the full implications must be understood and
   carefully weighed before choosing a different course.

Reply:
In fact, if the logo is visible by some contrasting parts, it should be sufficient if the text inside is given by an alternative like a tooltip. Where the tooltip should also explain the functionality.

Reply:
note that generally, you're not allowed to modify logos along the lines of "just put an outline around the logo's shapes"